src/react/outline.md

```diff
@@ -0,0 +1,78 @@
+@page react/outline React Content Outline
+@parent react 1
+
+@body
+
+## Intro
+
+* Before you begin
+  * Who is this guide for?
+    * Know the basics of Javascript, including ES2016 (include brief review)
+    * Familiar with NPM (include brief review)
+    * Want to build amazing frontend apps!
+* Create React App
+  * Why? What problem does it solve?
+    * discuss build systems in general
+  * Philosophy
+  * Ejecting
+* What is React
+  * a view framework
+  * an abstraction for the DOM (why?)
+* What is React **Not**
+  * a comprehensive application framework
+    * you *can* use it as one, but there are some additional libraries you should use for maximum effect
+  * What is it missing?
+    * state management (not technically, but here's why)
+    * model/api layer
+* What is JSX
+  * How it differs from HTML
+    * always close elements
+    * a couple props different
+  * Do I have to? No, but you really should.
+* The React Way
+  * "Everything is a Component"
+  * Progression from megalith to render components
```

Comment:
I don't know what this means. What is a megalith? Render components?

Reply:
this is the progression that you and i put together for the bitovi training. It goes from the long single component (megalith) to a better one using semantic components (but the main component is still just as long) to a super clean one that is less than half the size